test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -131,7 +131,7 @@ def test_pytorch_inference_torchdata_gpu(
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_CPU_INSTANCE_TYPE, indirect=True)
 def test_pytorch_inference_torchdata_cpu(pytorch_inference, ec2_connection, cpu_only, pt111_and_above_only):
     _, image_framework_version = get_framework_and_version_from_tag(pytorch_inference)
-    if Version(image_framework_version) in SpecifierSet(">=1.11,<=1.13"):
+    if Version(image_framework_version) in SpecifierSet(">=1.11,<=1.14"):
```

Comment:
In my previous comment, when I said upper bound, I meant `">=1.11,<1.14"` - let's remove the less than or equal to, even though we don't expect there to be a PT 1.14

Reply:
Updating